Source/Core/EllipsoidGeometry.js

```diff
@@ -50,6 +50,11 @@ define([
      *
      * @param {Object} [options] Object with the following properties:
      * @param {Cartesian3} [options.radii=Cartesian3(1.0, 1.0, 1.0)] The radii of the ellipsoid in the x, y, and z directions.
+     * @param {Cartesian3} [options.innerRadii=options.radii] The inner radii of the ellipsoid in the x, y, and z directions.
```

Comment:
Can you also mention updates to this and `EllipsoidOutlineGeometry` in CHANGES.md?